vfs/file.go

```diff
@@ -466,7 +479,17 @@ func (f *File) setObject(o fs.Object) {
 // the directory cache
 func (f *File) setObjectNoUpdate(o fs.Object) {
 	f.mu.Lock()
+	oldMode := f.mode
 	f.o = o
+	// FIXME: Is this needed at all ?
```

Comment:
I would have hoped that this isn't needed. It justs writesback the rclone object into the VFS object - it shouldn't change the handling of the VFS object.

Reply:
Ok, i will make some testing removing this code

Reply:
Tests still pass without that, so reverted.